content/workshops/ansible_tower/exercise1.4.adoc

```diff
@@ -115,15 +132,21 @@ You can manage playbooks and playbook directories by either placing them manuall
 under the Project Base Path on your Tower server, or by placing your playbooks into
 a source code management (SCM) system supported by Tower, including Git, Subversion, and Mercurial.
 
-*Step 1:* Click on PROJECTS +
+=== Step 1:
 
-*Step 2:* Select ADD     image:at_add.png[Add,35,25] +
+Click on PROJECTS +
 
-*Step 3:* Complete the form using the following entries +
+=== Step 2:
+
+Select ADD     image:at_add.png[Add,35,25] +
+
+=== Step 3:
+
+Complete the form using the following entries +
```

Comment:
You don't need the trailing `+` here. The extra line below this forces the new line.